src/main/java/com/weaverplatform/importer/xmi/ImportXmi.java

```diff
@@ -283,6 +316,53 @@ private String toCamelCase(String str) {
 
   /**
    * Creates an Weaver Individual
+   *
+   * @param subType
+   * @param superType
+   * @return
+   */
+  public void toWeaverGeneralization(String subType, String superType) {
+    
+    Entity subEntity = weaver.get(subType);
+    Entity superEntity = weaver.get(superType);
+    
+    if(EntityType.INDIVIDUAL.equals(subEntity.getType()) && EntityType.INDIVIDUAL.equals(superEntity.getType())) {
+
+      Entity annotations = weaver.get(subEntity.getRelations().get(RelationKeys.ANNOTATIONS).getId());
+      Entity subClassAnnotation = null;
+      for(String key : annotations.getRelations().keySet()) {
+        Entity candidateAnnotation = weaver.get(annotations.getRelations().get(key).getId());
+        if(candidateAnnotation != null && "rdfs:subClassOf".equals(candidateAnnotation.getAttributeValue("label"))) {
+          subClassAnnotation = candidateAnnotation;
+        }
+      }
+      Entity properties = weaver.get(subEntity.getRelations().get(RelationKeys.PROPERTIES).getId());    
+      
+      if(subClassAnnotation == null || properties == null) {
+        throw new RuntimeException("problem finding annotations or properties with generalization of "+subType+" --> "+superType);
+      }
+        
+      Map<String, ShallowEntity> relations = new HashMap<>();
+      relations.put("subject", subEntity);
+      relations.put("object", superEntity);
+      relations.put("annotation", subClassAnnotation);
+
+      HashMap<String, Object> propertyAttributes = new HashMap<>();
+      propertyAttributes.put("predicate", "rdfs:subClassOf");
+
+      Entity nameProperty = weaver.add(propertyAttributes, EntityType.INDIVIDUAL_PROPERTY, UUID.randomUUID().toString(), relations);
+
+
+      properties.linkEntity(nameProperty.getId(), nameProperty);
+      
+    } else {
+      throw new RuntimeException("skipping creation of generalization of "+subType+" --> "+superType);
```

Comment:
Throwing a runtime exception is not so much skipping as it is aborting the entire run. 

If skipping is intended, just log the message and move on
If aborting the run is intended: remove skipping from the message